### DISCUSSION TITLE
How can I deliver auto updates?

### DISCUSSION THREAD

**Discussion by umairtipu** [quasarframework/quasar#10954] [CLI - Electron mode]
I want to deliver auto update from github repo to the application user. Can anyone guide me how can I do this?

**Comment by MilosPaunovic** [comment]
Take a look at a GithHub Actions
https://github.com/features/actions

**Reply by umairtipu** [reply]
How a user will get to know when new update is available?

**Reply by MilosPaunovic** [reply]
Ummm... The user will see new content on the page.

**Reply by umairtipu** [reply]
we need to allow user either he/she wants to update the app or not

**Reply by MilosPaunovic** [reply]
So you want a dialog of some kind which will say "Hey, we have new content, refresh the page if you want to see it"?

**Reply by umairtipu** [reply]
Yes, actually the app will be updated and restarted when user click on 'Yes'. This is electron application.

**Reply by MilosPaunovic** [reply]
Ohh, okay, wasn't aware that it's about Electron application. I don't have much experience developing those, so someone else will have to take a look and help you.

**Reply by umairtipu** [reply]
Thank you for your time.

**Comment by hawkeye64** [comment]
I don't know much about this topic, but with a bit of searching I found a GitHub repo with an example of how to use Electron Autoupdate feature: https://github.com/iffy/electron-autoupdate-example
